anvil/src/winit.rs

```diff
@@ -212,8 +217,8 @@ pub fn run_winit() {
     let mut pointer_element = PointerElement::<GlesTexture>::default();
 
     while state.running.load(Ordering::SeqCst) {
-        if winit
-            .dispatch_new_events(|event| match event {
+        for event in winit.get_events() {
```

Comment:
I think we should move anvil over the an event source to match the udev and x11 backends. Plus the end goal is to have an event source from winit.

Reply:
You're free to do anvil yourself.